File: backend/ArduinoThermoHygrometer/DTOs/HumidityDto.cs
namespace ArduinoThermoHygrometer.Api.DTOs;

public class HumidityDto
{
    public Guid Id { get; set; } = default!;

    public DateTimeOffset RegisteredAt { get; set; } = default!;

    public decimal AirHumidity { get; set; } = default!;
}

File: api/src/main/java/api/arduinothermohygrometer/dtos/HumidityDto.java
package api.arduinothermohygrometer.dtos;

import java.time.LocalDateTime;

import jakarta.validation.constraints.DecimalMax;
import jakarta.validation.constraints.DecimalMin;
import jakarta.validation.constraints.NotNull;
import lombok.Builder;

@Builder
public record HumidityDto(@NotNull LocalDateTime registeredAt,
                          @DecimalMin("20.00") @DecimalMax("90.00") Double airHumidity
) {}
